controllers/machinedeployment_sync.go

```diff
@@ -34,13 +34,13 @@ import (
 	"k8s.io/client-go/util/retry"
 	"k8s.io/klog"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha2"
-	dutil "sigs.k8s.io/cluster-api/pkg/controller/machinedeployment/util"
+	dutil "sigs.k8s.io/cluster-api/controllers/mdutil"
```

Comment:
same here with the alias

Reply:
Renamed in the latest commit, good catch!